core/client/app/routes/settings/pass-protect.js

```diff
@@ -0,0 +1,44 @@
+import AuthenticatedRoute from 'ghost/routes/authenticated';
+import loadingIndicator from 'ghost/mixins/loading-indicator';
+import CurrentUserSettings from 'ghost/mixins/current-user-settings';
+import styleBody from 'ghost/mixins/style-body';
+
+var SettingsPassProtectRoute = AuthenticatedRoute.extend(styleBody, loadingIndicator, CurrentUserSettings, {
+	classNames: ['settings-view-pass'],
+
+	beforeModel: function () {
+        var feature = this.controllerFor('feature'),
```

Comment:
`controllerFor` is a deprecated macro for injecting a controller as a computed property, not for getting it in a method :)
you want something more like this:
```js
feature: Ember.inject.controller(),
beforeModel: function () {
  var feature = this.get('feature');
  ....
}
```